docs/content/usage/front-matter.mdx

```diff
@@ -38,13 +38,12 @@ status: Stable
 
 Doctocat comes with a few predefined statuses that have colors associated with them:
 
-| Status         | Output                                |
-| -------------- | ------------------------------------- |
-| `Stable`       | <StatusLabel status="Stable" />       |
-| `New`          | <StatusLabel status="New" />          |
-| `Experimental` | <StatusLabel status="Experimental" /> |
-| `Review`       | <StatusLabel status="Review" />       |
-| `Deprecated`   | <StatusLabel status="Deprecated" />   |
+| Status         | Output                                | Meaning                                                                |
+| -------------- | ------------------------------------- |------------------------------------------------------------------------|
+| `Alpha`        | <StatusLabel status="Alpha" />        | Experimental. Usage requires review from a Design Systems team member. |
+| `Beta`         | <StatusLabel status="Beta" />         | Usage encouraged, but not required.                                    |
+| `Stable`       | <StatusLabel status="Stable" />       | Usage required.                                                        |
```

Comment:
What does it mean for usage to be "required"?

Reply:
This is more specific to the company, than the broader community, I think. It means we intend for developers to use the component when it's available instead of using primer css.

I think perhaps this

```suggestion
| `Stable`       | <StatusLabel status="Stable" />       | The component is expected to not have breaking changes and you should use it!                                                        |
```

Reply:
@joelhawksley ^

Reply:
Ah, that's a good point. Maybe something like `Usage expected` would make sense?

Reply:
```suggestion
| `Stable`       | <StatusLabel status="Stable" />       | Usage expected.                                                        |
```